CLAUDE.md

```
@@ -1,851 +1,98 @@
-# CLAUDE.md
-
-This file provides guidance to Claude Code when working with the BSMarker codebase.
-
-# BSMarker - Bird Song Annotation Tool
+# CLAUDE.md - BSMarker
 
 ## Test Credentials
+- **URL**: https://bsmarker.utia.cas.cz
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)